src/bidiMapper/modules/context/BrowsingContextImpl.ts

```diff
@@ -712,6 +712,7 @@ export class BrowsingContextImpl {
       case BrowsingContext.UserPromptType.Beforeunload:
         return (
           this.#unhandledPromptBehavior?.beforeUnload ??
+          this.#unhandledPromptBehavior?.default ??
           // In WebDriver Classic spec, `beforeUnload` prompt should be accepted by
```

Comment:
This comment is not correct as well

Reply:
Can you tell me what exactly it should be? As you added it and context is skipping me atm.

Reply:
oh, the comment is about the different default. Not problem